api/openapi-spec/swagger.json

```
@@ -10809,12 +10809,14 @@
     ],
     "properties": {
      "externalResourceProvider": {
+      "description": "The name of the resource that is representing the device. Exposed by a device plugin and requested by VMs. Typically of the form vendor.com/product_nameThe name of the resource that is representing the device. Exposed by a device plugin and requested by VMs. Typically of the form vendor.com/product_name",
       "type": "boolean"
      },
      "pciVendorSelector": {
       "type": "string"
      },
      "resourceName": {
+      "description": "\"The vendor_id:product_id tupple of the PCI device",
```

Why do you need the `\"`? Not sure if it is breaking things.
```suggestion
      "description": "The vendor_id:product_id tuple of the PCI device",
```